config.yaml

```diff
@@ -99,7 +99,7 @@ matrix:
           - antiddos_skip_bad_type
           - as
           - as_skip_v1
-          - bms_skip
+          - bms
```

Comment:
also remove bms_skip entries from eu-nl to eu-nl and from eu-de to eu-nl monitoring scopes

Reply:
The requested changes are done, please review again.